Extension/browser/webkit/webrequest.js

```diff
@@ -116,11 +116,7 @@ function onHeadersReceived(requestDetails) {
          */
         var websocketCheckUrl = "ws://adguardwebsocket.check/";
         if (webRequestService.checkWebSocketRequest(tab, websocketCheckUrl, referrerUrl)) {
-
-            //We don't change connect-src directives for older chromes
-            //https://github.com/AdguardTeam/AdguardBrowserExtension/issues/385
-            var blockConnectSrc = Prefs.chromeVersion > 51;
-            if (CspUtils.blockWebSockets(responseHeaders, !blockConnectSrc)) {
+            if (CspUtils.blockWebSockets(responseHeaders)) {

```

Comment:
One more thing. Instead of changing existing CSP header we should add a new one.
`A website can declare multiple CSP headers, also mixing enforcement and report-only ones. Each header will be processed separately by the browser.`

So simply push a new header with value:
`frame-src http: https:; child-src http: https:`

We can get rid of CspUtils after that:)